### PR TITLE
feat(InfluxQL): add doc on Influx query and on databases

### PR DESCRIPTION
**Pull request by aurrelhebert**

### CODE REVIEW

File: pages/platform/metrics/protocol_influxdb/guide.en-gb.md [hunk 1]
```diff
@@ -243,6 +252,33 @@ Example:
...
+  --url 'https://m:READ_TOKEN@influxdb.gra1.metrics.ovh.net/query?q=SELECT%20%22used_percent%22%20FROM%20%22disk%22%20WHERE%20%20time%20%3E%3D%20now()%20-%2020m&=%20'
+```
+
+This will execute the following InfluxQL query:
+
+```InfluxQL
+SELECT "used_percent" FROM "disk" WHERE  time >= now() - 20m
+```
+
+For the one used to query Influx, you will notify that the `db` mandatory parameter of Influx isn't set in this query. With Metrics the database field is optionnal, as Metris doesn't rely on Data-bases to store its metrics. If you need segmentation, you can use different Metrics project or isolate with an additional label.
```
Comment (Alkorin) [inline pages/platform/metrics/protocol_influxdb/guide.en-gb.md:270]:
```suggestion
For the one used to query Influx, you will notice that the `db` mandatory parameter of Influx is not set in this query. With Metrics the database field is optional, as Metrics does not rely on databases to store its metrics. If you need segmentation, you can use different Metrics project or isolate with an additional label.
```

File: pages/platform/metrics/protocol_influxdb/guide.en-gb.md [hunk 2]
```diff
@@ -198,6 +203,8 @@ The existing [SHOW statements](https://docs.influxdata.com/influxdb/v1.7/query_l
 | SHOW SERIES             | <i class="fas fa-check"></i> |
 | SHOW TAG VALUES         | <i class="fas fa-check"></i> |
 
+As in Metrics, the concept of DataBases doesn't exists, the `SHOW DATABASES` statement will always return only one database: `metrics`.
```
Comment (Alkorin) [inline pages/platform/metrics/protocol_influxdb/guide.en-gb.md:206]:
```suggestion
As the concept of databases does not exists in Metrics, the `SHOW DATABASES` statement will always return only one database: `metrics`. 
```

File: pages/platform/metrics/protocol_influxdb/guide.en-gb.md [hunk 3]
```diff
@@ -214,6 +221,8 @@ The existing [database management statements](https://docs.influxdata.com/influx
 | ALTER RETENTION POLICY | <i class="fas fa-times"></i> |
 | DROP RETENTION POLICY | <i class="fas fa-times"></i> |
 
+As the `CREATE DATABASE` statement is used in some client, this statement was implemented in Metrics and always return. However no databases exists in Metrics.
```
Comment (Alkorin) [inline pages/platform/metrics/protocol_influxdb/guide.en-gb.md:224]:
```suggestion
As the `CREATE DATABASE` statement is used by some client, this statement was implemented in Metrics and always return. However no database exists in Metrics.
```

File: pages/platform/metrics/protocol_influxdb/guide.en-gb.md [hunk 4]
```diff
@@ -243,6 +252,33 @@ Example:
...
+```InfluxQL
+SELECT "used_percent" FROM "disk" WHERE  time >= now() - 20m
+```
+
+For the one used to query Influx, you will notify that the `db` mandatory parameter of Influx isn't set in this query. With Metrics the database field is optionnal, as Metris doesn't rely on Data-bases to store its metrics. If you need segmentation, you can use different Metrics project or isolate with an additional label.
+
+### Set up InfluxDB on Grafana
+
+Then select `basic auth` and fill the **user** with a non empty value, `metrics` for example.
+The basic auth password need to be valid Metrics `READ_TOKEN`.
```
Comment (Alkorin) [inline pages/platform/metrics/protocol_influxdb/guide.en-gb.md:275]:
```suggestion
The password needs to be a valid Metrics `READ_TOKEN`.
```

File: pages/platform/metrics/protocol_influxdb/guide.en-gb.md [hunk 5]
```diff
@@ -243,6 +252,33 @@ Example:
...
+Please ensure that you have set the `metrics` key in the database field as describe in the screen below.
```
Comment (Alkorin) [inline pages/platform/metrics/protocol_influxdb/guide.en-gb.md:276]:
```suggestion
Please ensure that you have set the `metrics` key in the database field as described in the screen below.
```

File: pages/platform/metrics/protocol_influxdb/guide.en-gb.md [hunk 6]
```diff
@@ -243,6 +252,33 @@ Example:
...
+
+![menu](images/grafana-datasource-influx.png){.thumbnail}
+
+Your Influx datasource is now set and ready to be query.
```
Comment (Alkorin) [inline pages/platform/metrics/protocol_influxdb/guide.en-gb.md:280]:
```suggestion
Your Influx datasource is now set and ready to be queried.
```

**Comment by OVH-Team-Guides** [comment]
Merge is done, thanks for your contribution.

**Pull request merged**